cookietemple/lint/template_linter.py

```diff
@@ -649,6 +661,43 @@ def check_missing_sections(self, parsed_sections) -> bool:
             self.linter_ctx.passed.append(('general-7', 'All required cookietemple.cfg sections were found!'))
             return True
 
+    def lint_sync_blacklisted_section(self, section_items) -> bool:
```

Comment:
Decided to put this logic into a separate function, since it differs from the other section linting more